maqs-base/src/test/java/com/cognizantsoftvision/maqs/base/BaseTestObjectUnitTest.java

```diff
@@ -277,27 +358,16 @@ public void testContainsAssociatedFile() {
     } catch (IOException e) {
       e.printStackTrace();
     }
-    assert temp.exists();
+
+    Assert.assertNotNull(temp);
+    Assert.assertTrue(temp.exists());
+
     final String path = temp.getAbsolutePath();
     Assert.assertTrue(testObject.addAssociatedFile(path), "Checking that associated file was added");
     Assert.assertNotNull(testObject.getArrayOfAssociatedFiles(), "Checking that array is instantiated");
     Assert.assertTrue(testObject.containsAssociatedFile(path), "Checking if array contains file");
   }
 
-  // Test Setup Objects
```

Comment:
Did you mean to delete the entire summary?

Reply:
yes, this code has been commented out for a couple years. I figured it was time to bite the bullet and remove it.